### DISCUSSION TITLE
Overhaul genetic materials

### DISCUSSION THREAD

**Discussion by 8on8er** [FakeFishGames/Barotrauma#12135] [Ideas and suggestions]
Having played many campaigns the recurring problem of genetic material gathering being too grindy and burdensome and random prevails. I often finish the campaign and give up searching for genetic material because it's just not necessary and requires too much effort with the drops being so random.

I have two main suggestions:

> The genetic material you find when looting corpses directly corresponds to the monster you are looting.
For example if you would like to farm and gain lots of mudraptor genes, then you will have to kill and hunt lots of mudraptors to get that gene. Also increase the drop %.

> The second suggestion is introduce genes for the abyssal creatures.
- Latcher - allows you to shoot a tongue at creatures like a harpoon. Essentially functions the same as a harpoon gun.
- Charbarjryeuey - I can never remember how to spell it - the demon lion dolphin.
- you get the old security officer swim dash perk, maybe for the sake of balance make it have a generally long recharge time compared to previously?
- Doom worm - I have no idea to be honest what gene equivalent would work for the doom worm? Maybe regenerating body armour that can be destroyed like the worm itself? Not just a % damage reduction or resistance but armour like a mudraptor that can be broken and won't repair until the next round however I'm all for ideas for this one.

All this said why I believe this would be good:

> the first change would mean genetic material is no longer a mindless lottery and grind with varying to no results. You would actually have the possibility of getting some good genes with work before you complete the campaign and YOU could choose specifically which genes, if you want hammerhead genes you are going to have to hunt hammerheads. If you want a latcher tongue? You are going to have to kill 10 latchers. It would make players explore and seek out combat more.

> the second change again I believe would encourage players to not ignore abyssal monsters but to seek them out and hunt them like moby dick

Let me know what you think sorry if this seems unhinged I'm writing this as I'm walking because I was thinking about it.

**Comment by Rotiart-Drol** [comment]
Honestly the framework for genetic material is almost completely satisfactory for me as-is, the one and only gripe I have is gene quality. I'm not going to beat around the bush, I think gene quality should just be retired completely. It already clashes with an enormous chunk of genes, like spineline, husk, mudraptor, and matriarch, and for almost all the other genes it's far too punishing a grind for what you receive. Genes like Mantis, Mollusc, and Skitter gives bonuses that don't even come close to comparing with the average talent, while being way more grindy to obtain, and able to be lost on death.

The problem with removing gene quality? Moloch Genes. And honestly I'm not even sure I'd chalk it up to the gene itself being problematic. The game just rewards you for doubling down on damage resistance at every turn like a monkey. Damage resistance is a can of worms on it's own, but it does intersect with the genetic material balancing debate

**Reply by 8on8er** [reply]
> Honestly the framework for genetic material is almost completely satisfactory for me as-is, the one and only gripe I have is gene quality. I'm not going to beat around the bush, I think gene quality should just be retired completely. It already clashes with an enormous chunk of genes, like spineline, husk, mudraptor, and matriarch, and for almost all the other genes it's far too punishing a grind for what you receive. Genes like Mantis, Mollusc, and Skitter gives bonuses that don't even come close to comparing with the average talent, while being way more grindy to obtain, and able to be lost on death.
> 
> The problem with removing gene quality? Moloch Genes. And honestly I'm not even sure I'd chalk it up to the gene itself being problematic. The game just rewards you for doubling down on damage resistance at every turn like a monkey. Damage resistance is a can of worms on it's own, but it does intersect with the genetic material balancing debate

Thank you for expressing you feel harvesting genes is not too grindy however with your second point I feel the argument for removing gene quality doesn't pertain to the changes suggested and warrants a suggestion thread of it's own.

**Reply by JoneKone** [reply]
.. It can be grind so you have some busy work on down time.. and when seas are calm. Or the team is diving and sec is keeping ship in shape. etc etc. 

Also you could get a computer for genetic material analytics and captain can upgrade set computer to make it more faster. "Combine fragmented dna/rna from incomplete specimens. And lets say there is a monster inside your sub just by change somebody kills it and you could possibly extract in a shake free environment (on deck) a bone narrow sample directly from the spine? And lets say xD the specimen was poisoned so no bullets destroyed the bone narrow... suddenly making a poison very good option kill a specimen... Yees More complexity creates more possibilities and more opportunities.

**Comment by BiffW** [comment]
I want to mention a discussion I posted earlier about making unresearched genetic material stackable. It makes no sense to keep them unstackable since every unresearched genetic material is the same. Just make them stack up to 8 like every other material.
https://github.com/Regalis11/Barotrauma/discussions/11424#discussion-4916780

**Comment by Anarengon** [comment]
Before I'd suggest overhauling the whole gene system, I'd suggest just doing some buffs and nerfs to each type of gene. There's 12 types, 4 are embarrassingly useless, and 1 has been recently outclassed by "wearing a certain shirt". I'd vote against adding special genes from abyss creatures, because I think having wide power swings, such as between "did manage to catch the abyss gene" and "didn't" is bad for game balance.
